dev-libs/libutf8proc/libutf8proc-2.2.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+MY_PN="${PN/lib/}"
+
+inherit cmake-utils toolchain-funcs
+
+DESCRIPTION="A C library that provides various operations for data in the UTF-8 encoding"
+HOMEPAGE="https://juliastrings.github.io/utf8proc/"
+SRC_URI="https://github.com/JuliaStrings/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="test"
+
+# Tests do need internet access
+RESTRICT="test"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+DOCS=(
+	"lump.md"
+	"NEWS.md"
+	"README.md"
+)
+
+PATCHES=( "${FILESDIR}/${P}-fix-path.patch" )
+
+src_prepare() {
+	cmake-utils_src_prepare
```

Comment:
call `tc-export AR CC` then I believe src_compile() can be removed

Reply:
Yes, this works. Thanks!